### ISSUE TITLE
Please provide Wiki details for Sickbeard paragraph on API details in AutoConfiguration.cfg

### ISSUE THREAD

**Issue by Xenocell12** [clinton-hall/nzbToMedia#1379]
Hello!

Well, I had the most nightmarish experience trying to figure out WHY Medusa (Sickbeard Fork) wasn't seeing any directories to process or really respond to the nzbtomedia scripts.

Turns out I was providing too much information for Medusa and thus things were being canceled out.

Credentials are only required at least for Medusa. Haven't tried API only. 

Question, which do you reccomend? What is the purpose of the API section for Sickbeard?

TLDR: I was a dummy and didn't need API filled in for Medusa if I was providing credentials. Once removed, everything started working again!

Hope none else had the same experience as me. 🤣🤣🤣

**Comment by Xenocell12** [comment]
Found it.

echel0n added this but failed to mention the addition in wiki. However, its obvious in the code its for SiCKRAGE and only that fork.

No wonder it breaks Medusa so bad. (This guys has huge beef with Medusa teams)

https://github.com/clinton-hall/nzbToMedia/commit/5f2a5499b8aa763f483068bb11d16c92c5f55fbc#diff-0ca1b562d9105f64108367bb6dd7e709

**Comment by Xenocell12** [comment]
Updated wiki.

**Comment by clinton-hall** [comment]
thanks for updating wiki.

From my side, I had added "api" to the options a while ago as there were talks from various sickbeard forks  about moving to api... but once I added this I discovered that the api's didn't support the rename/postprocess function.

echelon pushed this once he added api postprocessing. I hadn't realised this was an issue for other branches, as on my setup I didn't have the api configured.
